src/functions/template-tags/date.php

```diff
@@ -342,6 +349,13 @@ function tribe_get_end_date( $event = null, $display_time = true, $date_format =
 			return '';
 		}
 
+		static $end_dates = [];
+		$cache_key = "{$event->ID}:{$display_time}:{$date_format}:{$timezone}";
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: a64ce6f9dab8c7d4fae9d23802e589e0</sub>